Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj

```diff
@@ -1,7 +1,4 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
-  <PropertyGroup Condition="$(TargetFramework) == 'uap10.0.14393'">
```

Comment:
Intentional?

Reply:
Yea there's already code that sets this constant here

https://github.com/xamarin/Xamarin.Forms/blob/master/UWP.Build.props#L9